### ISSUE TITLE
[BUG] Create timeline visualization is broken in playground 2.12

### ISSUE THREAD

**Issue by seraphjiang** [opensearch-project/OpenSearch-Dashboards#6243]
**Describe the bug**

Create timeline visualization is broken in playground 2.12

**To Reproduce**
1. Go to 'https://playground.opensearch.org/app/visualize#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))'
2. Click on 'Create visualization'
3. Choose 'timeline'
https://playground.opensearch.org/app/visualize#/create?type=timelion&_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))&_a=(filters:!(),linked:!f,query:(language:kuery,query:''),uiState:(),vis:(aggs:!(),params:(expression:'.opensearch(*)',interval:auto),title:'',type:timelion))

```
Error
Error: Illegal value for token color: #F66
    at e.getId (https://playground.opensearch.org/221231/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:387:1468419)
    at Do (https://playground.opensearch.org/221231/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:387:1468134)
    at e.createFromParsedTokenTheme (https://playground.opensearch.org/221231/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:387:1469351)
    at e.createFromRawTokenTheme (https://playground.opensearch.org/221231/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:387:1468825)
    at e.get [as tokenTheme] (https://playground.opensearch.org/221231/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:387:1478490)
    at e.setTheme (https://playground.opensearch.org/221231/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:387:1480020)
    at new t (https://playground.opensearch.org/221231/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:387:1437763)
    at https://playground.opensearch.org/221231/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:387:1532449
    at qa (https://playground.opensearch.org/221231/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:387:1532360)
    at Object.Ra [as create] (https://playground.opensearch.org/221231/bundles/osd-ui-shared-deps/osd-ui-shared-deps.js:387:1532418)
```

**Expected behavior**
1. be able to create timeline viz
2. add some automation test to cover 

**OpenSearch Version**
2.12

**Dashboards Version**
confirmed 2.12, not sure about previous version

**Plugins**

all (MDS enabled with local cluster)

**Screenshots**

![image](https://github.com/opensearch-project/OpenSearch-Dashboards/assets/550437/11041b93-03a9-4fea-8695-91c6bb74fd85)

**Comment by AMoo-Miki** [comment]
The version of `monaco-editor` OSD uses only accepts 6+2 hex color codes. The next-dark theme of OUI has one 3-letter hex code which is breaking the experience.

**Comment by zhyuanqi** [comment]
Hi @AMoo-Miki  Are we going to have any fix on it?

**Comment by AMoo-Miki** [comment]
According to [OUI CHANGELOG](https://github.com/opensearch-project/oui/blob/main/CHANGELOG.md#160), a fix went our in their 1.6.0 release. However, OSD only employed OUI@1.8.1 in their 2.16 release.
